docs/user-guide/rules.md

```diff
@@ -240,6 +240,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`no-browser-hacks`](../../src/rules/no-browser-hacks/README.md): Disallow browser hacks that are irrelevant to the browsers you are targeting.
 - [`no-descending-specificity`](../../src/rules/no-descending-specificity/README.md): Disallow selectors of lower specificity from coming after overriding selectors of higher specificity.
 - [`no-duplicate-selectors`](../../src/rules/no-duplicate-selectors/README.md): Disallow duplicate selectors.
+- [`no-empty-source`](../../src/rules/no-empty-source/README.md): Disallow empty source.
```

Comment:
It's really nice to see the old English teacher inside you come out and take the time to explain these little peculiar language things :)